RadixWallet/EngineKit/RETDecimal+Extensions.swift

```diff
@@ -246,7 +246,23 @@ extension RETDecimal {
 		let integerCount = UInt(Swift.max(digits.count - RETDecimal.scale, 1))
 		if integerCount > totalPlaces {
 			let scale = RETDecimal(exponent: integerCount - totalPlaces)
-			return (self / scale).rounded(decimalPlaces: 0) * scale
+			let base = self / scale
```

Comment:
I would say that we don't want to protect ourselves against any kind of error here, only the specific case of max, no? We could just

```
guard self != max else {
  // return the number rounded towards zero
}
```

Reply:
No because if the value is close to max we don't want to fail either. Hmm